google/cloud/storage/internal/rest_client.cc

```diff
@@ -31,6 +31,7 @@
 #include "google/cloud/internal/getenv.h"
 #include "absl/memory/memory.h"
 #include "absl/strings/match.h"
+#include "absl/strings/strip.h"
```

Comment:
FYI, this is a new header from Abseil, keep a weather eye on the builds.  MSVC may break.